packages/react-ui/src/theme/sections/typography.js

```diff
@@ -9,117 +11,219 @@ export const themeTypography = {
   },
   button: {
-    fontFamily: 'Montserrat, sans-serif',
-    fontWeight: 600,
-    fontSize: '0.875rem',
-    lineHeight: 1.715,
-    letterSpacing: '0.018em',
-    textTransform: 'inherit',
-    fontSmoothing: 'antialiased'
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(13),
+    lineHeight: 1.538,
+    letterSpacing: '0.25px',
+    textTransform: 'inherit'
   },
   caption: {
-    fontFamily: '"Open Sans", sans-serif',
-    fontWeight: 600,
-    fontSize: '0.75rem',
-    lineHeight: 1.334,
-    letterSpacing: '0.017em',
-    fontSmoothing: 'antialiased'
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 400,
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.2px'
   },
   overline: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(10),
+    lineHeight: 1.2,
+    letterSpacing: '2px',
+    textTransform: 'uppercase'
+  }
+};
+
+const customTypography = {
+  body1Italic: {
+    ...baseTypography.body1,
+    fontStyle: 'italic'
+  },
+  body1Strong: {
+    ...baseTypography.body1,
+    fontWeight: 600
+  },
+  body1StrongItalic: {
+    ...baseTypography.body1,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  body2Italic: {
+    ...baseTypography.body2,
+    fontStyle: 'italic'
+  },
+  body2Strong: {
+    ...baseTypography.body2,
+    fontWeight: 600
+  },
+  body2StrongItalic: {
+    ...baseTypography.body2,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  buttonLarge: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: '0.25px'
+  },
+  buttonSmall: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.4px'
+  },
+  captionItalic: {
+    ...baseTypography.caption,
+    fontStyle: 'italic'
+  },
+  captionStrong: {
+    ...baseTypography.caption,
+    fontWeight: 500
+  },
+  captionStrongItalic: {
+    ...baseTypography.caption,
+    fontWeight: 500,
+    fontStyle: 'italic'
+  },
+  overlineDelicate: {
+    ...baseTypography.overline,
+    fontWeight: 400,
+    letterSpacing: '1.2px'
+  },
+  codeLarge: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 400,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeLargeStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeMedium: {
+    fontFamily: '"Overpass Mono", monospace',
     fontWeight: 400,
-    fontSize: '0.625rem',
-    lineHeight: 1.6,
-    letterSpacing: '0.150em',
-    textTransform: 'uppercase',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(14),
+    lineHeight: 1.428,
+    letterSpacing: 0
+  },
+  codeMediumStrong: {
```

Comment:
If we're gonna go after several 'nested' concepts here, should we create a proper object (to use keyed names)?

I'm thinking about something similar to (pseudo-code):

```
code : { 
     default: {  /* style definition */ }
     large: {
          default: { /*whatever*/}
          strong: { ... }   
     }
}
```

So we would consume externally something like: `code.large.strong`

Also I'm feeling that this set of custom elements could be maybe separated from official typography in the tree, having like a first level with `custom : { /* previous keys here */ }` Would it be too verbose? or complex? What do you think?

Reply:
Good point. I like what you're proposing about nesting, I'll give it a try.